code/modules/clothing/gloves/insulated.dm

```diff
@@ -12,6 +12,10 @@
 	resistance_flags = NONE
 	custom_price = PAYCHECK_CREW * 10
 	custom_premium_price = PAYCHECK_COMMAND * 6
+	cold_protection = HANDS
+	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
+	heat_protection = HANDS
+	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
```

Comment:
The temperature protection required for bulbs is 360.
`GLOVES_MAX_TEMP_PROTECT` is 1500.
Make a new define and use it both on these gloves and on lightbulbs.

Reply:
then I would also have to put the new definition on the rest of the gloves that stop lightbulb damage like black gloves and advance insulated gloves if there is a need for it I can do it but not sure why it wasn't already that way for those kinds of gloves

Reply:
Black and Advanced Insulated gloves are literally listed as "fire-resistant".